Applications/Spire/Source/Ui/DurationBox.cpp

```diff
@@ -268,6 +262,30 @@ namespace {
     }
   };
 
+  QValidator::State HourModel::set(const Type& value) {
```

Comment:
Implement this at the point of declaration, as with the other methods for HourModel.